### PR TITLE
Devtools and Cross platform accelerators

### PR DESCRIPTION
**Pull request by priyansh1161**

### CODE REVIEW

File: src/login.html [hunk 1]
```diff
@@ -14,9 +14,8 @@
         <label for="password">Password</label>
         <input type="password" id="password">
         <input type="submit" id="btn-login" value="Login">
+      </form>
```
Comment (ryzokuken) [inline src/login.html]: Could you please fix the indentation here? It seems a little off to me.

File: .gitignore [hunk 2]
```diff
@@ -35,6 +35,7 @@ build/Release
 # Dependency directories
 node_modules/
 jspm_packages/
+.idea/
```
Comment (ryzokuken) [inline .gitignore:38]: What is the use of this directory? Is it editor-specific?

**Review by ryzokuken — COMMENTED** [review]
This looks great, actually!

Please look into the two comments I left and then we will merge this.

**Comment by ryzokuken** [comment]
@priyansh1161 I willingly deleted the `package-lock.json` (I had been using npm earlier) and added the yarn lockfile instead. Should we use both lockfiles, or should we stick to yarn?

**Comment by priyansh1161** [comment]
I use `npm` instead of `yarn` but if, there are some contributing guidelines, do let me know, i'll follow them, however with npm 5 there is no real use of yarn left.

File: .gitignore (hunk 2 above)
Comment (priyansh1161) [reply .gitignore:38]: yes, webstrom specific

**Pull request merged**